node-syscall/syscall.cc

```diff
@@ -25,7 +25,17 @@ intptr_t toNative(Local<Value> value) {
     Local<Array> array = Local<Array>::Cast(value);
     intptr_t* native = reinterpret_cast<intptr_t*>(malloc(array->Length() * sizeof(intptr_t))); // TODO memory leak
     for (uint32_t i = 0; i < array->Length(); i++) {
+#if (NODE_MODULE_VERSION >= NODE_6_0_MODULE_VERSION)
+      v8::Local<v8::Value> elem = array->Get(i);
```

Comment:
This file does `using namespace v8;` on top, and 4 lines above we have `Local<Array> array`, etc.

So I think it'd be better to be consistent and drop the namespace here as well:

```C++
Local<Value> elem = array->Get(i);
```

Similarly on another line below.

Reply:
Fixed.